<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResConvertWidget</class>
 <widget class="QWidget" name="ResConvertWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>911</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_build">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>20</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>启动</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_allBranchesTip">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>53</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;思源黑体 CN Regular&quot;;</string>
   </property>
   <property name="text">
    <string>操作平台</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_resPlatform">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>890</width>
     <height>361</height>
    </rect>
   </property>
   <property name="title">
    <string>显示区</string>
   </property>
   <widget class="QListWidget" name="listWidget_processed">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>20</y>
      <width>400</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="DragListWidget" name="listWidget_pending">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>400</width>
      <height>331</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="dragEnabled">
     <bool>true</bool>
    </property>
    <property name="dragDropOverwriteMode">
     <bool>true</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SelectionMode::MultiSelection</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>100</y>
      <width>16</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string>已转文件列表</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>16</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string>待转文件列表</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_emptyPending">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>40</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>清空</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_emptyProcessed">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>10</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>清空</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_dropTips">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>160</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>从系统资源管理器中
拖拽资源至此</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>190</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>In</string>
   </property>
   <widget class="QLineEdit" name="lineEdit_inPath">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>输入路径</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_inPath">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>24</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>10</y>
     <width>190</width>
     <height>51</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>指定文件转换完成后的输出路径</string>
   </property>
   <property name="statusTip">
    <string>指定文件转换完成后的输出路径</string>
   </property>
   <property name="title">
    <string>Out1</string>
   </property>
   <widget class="QLineEdit" name="lineEdit_outPath">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>输出路径</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_outPath">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>24</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>10</y>
     <width>190</width>
     <height>51</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>相同的内容可以选定额外路径再输出一次</string>
   </property>
   <property name="statusTip">
    <string>相同的内容可以选定额外路径再输出一次</string>
   </property>
   <property name="title">
    <string>Out2</string>
   </property>
   <widget class="QLineEdit" name="lineEdit_outPath_extra">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>输出路径</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_outPath_extra">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>24</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DragListWidget</class>
   <extends>QListWidget</extends>
   <header>sample.qt.src.pyui.DragListWidget</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
